.ci/test-install-docs.sh

```diff
@@ -6,12 +6,23 @@
 
 set -e
 
+# The go binary isn't installed, but we checkout the repos to the standard
+# golang locations.
+export GOPATH=${GOPATH:-${HOME}/go}
+
 info()
 {
 	local msg="$*"
 	echo "INFO: $msg"
 }
 
+setup()
+{
+	source /etc/os-release || source /usr/lib/os-release
+
+	mkdir -p "${GOPATH}"
```

Comment:
I dunno if this will fail if `GOPATH` already exists? If so, a `|| true` type fix I guess?

Reply:
Actually, no, `-p` won't fail if it exists.